app/models/form/sales/subsections/household_characteristics.rb

```diff
@@ -32,7 +32,7 @@ def pages
       Form::Sales::Pages::RetirementValueCheck.new("age_2_buyer_retirement_value_check", nil, self, person_index: 2),
       Form::Sales::Pages::GenderIdentity2.new(nil, nil, self),
       Form::Sales::Pages::RetirementValueCheck.new("gender_2_buyer_retirement_value_check", nil, self, person_index: 2),
-      ethnic_pages_for_buyer_2,
+      pages_for_buyer_2,
```

Comment:
I think this new name is misleading - maybe `buyer_2_identity_pages`?  or just something like `buyer_2_ethnicity_and_nationality_pages`

Reply:
I am hardcore ambivalent about this. happy to change it